### DISCUSSION TITLE
Download Songs in Different Formats

### DISCUSSION THREAD

**Discussion by Mendungo** [pear-devs/pear-desktop#1255] [Ideas]
I have been using this app for a long time and it's fantastic actually the best out there! But it's missing what a bunch of other alternatives have which is the downloadable music format, currently only MP3 is available which is a lossy audio file, old and very poor quality, if possible, other formats such as MQA or FLAC ("lossless" formats) should be supported or even AAC which is "lossy" but better quality, that way some communities such as DJS can use this tool to download songs and use in their mixes.

Thanks

**Comment by ArjixWasTaken** [comment]
That's nice and all, but youtube only streams up to 256 kbps AAC, anything higher would be placebo because those bits are not streamed.
Using a flac file would not increase the quality of the audio, flac is a container, if the data in the container is lossy, then the container can't help it.

Supporting flac would be pointless.
Go use deezer if you want higher quality audio, but then you have a much smaller library of songs.

**Reply by ArjixWasTaken** [reply]
Spotify ain't much better, 160kbps for Spotify free, 320kbps of bitrate for Spotify Premium...

320kbps is the maximum bitrate an mp3 can hold...
So if you use any spotify downloader that promises higher quality, and flac downloads, that's placebo for you

Converting an mp3 to a flac does not increase the bitrate, that is impossible to achieve unless you use AI to fill-in the gaps

**Reply by ArjixWasTaken** [reply]
I'm saying all these because my mother is also a DJ and I am tired of this hoax.
You cannot download good quality audio off-youtube, all the download programs you use are not downloading a flac, they download audio encoded in AAC, or WEBA, and convert it to FLAC
